examples/hosts/app-integration/external-data/src/model/taskList.ts

```diff
@@ -79,42 +97,15 @@ export class TaskList extends DataObject implements ITaskList {
         // To add a task, we update the root SharedDirectory.  This way the change is propagated to all collaborators
         // and persisted.  In turn, this will trigger the "valueChanged" event and handleTaskAdded which will update
         // the this.tasks collection.
+
+        this.savedData.set(id, { id, name: nameString.handle, priority: priorityCell.handle });
+
         const data: PersistedTask = {
             id,
             name: nameString.handle as IFluidHandle<SharedString>,
```

Comment:
Now that we have two separate maps, we also need to store two separate cells and strings in them (otherwise, both will be pointing at the same `nameString`/`priorityCell` and modifying one will also modify the other).  So e.g. you'll want to create a `draftNameString` as well as a `savedNameString` above.